screens/ThermostatScreen.tsx

```diff
@@ -169,7 +169,7 @@ export function ThermostatScreen() {
                 </View>
                 <View style={styles.thermostatGauge}>
                   <ThermostatGauge
-                    label={data.mode === "auto" ? "Energy Efficient" : " "}

```

Comment:
Can you generate a screenshot of the old behavior here? We may want to figure out how to recreate that visual indicator since we're removing it here. This will be something to cover in the meeting Thursday

Reply:
Yes! Here's also a quick video of what the text looks like in different modes.

![Screenshot 2024-02-13 at 2 51 35 PM](https://github.com/ACE-IoT-Solutions/connecting-mha-app/assets/94999450/3d1b2135-1c4b-41a8-982a-bbc7641e4c93)

https://github.com/ACE-IoT-Solutions/connecting-mha-app/assets/94999450/22dde2d1-8d64-4f40-8b7a-cb3a9970c68c

Reply:
Great!